src/components/Search.js

```diff
@@ -3,12 +3,11 @@ import { ActivityIndicator } from "react-native";
 import Render_search_timeline from "./Render_search_timeline";
 import Render_search_map from "./Render_search_map";
 import { Container, Header, Input, Item, Icon, Text } from "native-base";
+import get from "lodash/get";
 
 class Search extends React.Component {
   constructor(props) {
     super(props);
-    const { load_user } = this.props;
-    load_user();
```

Comment:
I removed it as we don't want to trigger a fetch at the very beginning as text was `""` anyway